demos/simpleBuildAgainstTrilinos/CMakeLists.txt

```diff
@@ -33,11 +40,11 @@ set(CMAKE_C_FLAGS  "${Trilinos_C_COMPILER_FLAGS} ${CMAKE_C_FLAGS}")
 set(CMAKE_Fortran_FLAGS  "${Trilinos_Fortran_COMPILER_FLAGS} ${CMAKE_Fortran_FLAGS}")
 
 # End of setup and error checking.
-#
-# NOTE: PROJECT command checks for compilers, so this statement
-# is moved AFTER setting compilers.
 
-project(MyApp)
+# Now enable the compilers now that we have gotten them from Trilinos
+enable_language(C)
+enable_language(CXX)
+enable_language(Fortran)
```

Comment:
@bartlettroscoe - what if trilinos disables fortran support? Does this need to check before adding enable_language(fortran)?

Reply:
@rppawlo, please see the latest commit 8e68f635ff4bece1b27a5c008301ad3003af16c6.  If this is sufficient, please approve.